src/frontends/ir/tests/test_model_wrong_shape.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" ?>
```

Comment:
I propose to avoid models on the file system, we always can create these files in the runtime.

Reply:
how to create a model with wrong share? I suppose ngraph always validates shapes and it will be not easy to do ..

Reply:
We can serialize it to file from the string

Reply:
done